src/panels/logbook/ha-logbook.ts

```diff
@@ -148,13 +148,19 @@ export class HaLogbook extends LitElement {
       );
     }
 
-    this._logbookEntries = undefined;
     this._throttleGetLogbookEntries();
   }
 
-  protected updated(changedProps: PropertyValues): void {
-    super.updated(changedProps);
+  protected shouldUpdate(changedProps: PropertyValues): boolean {
```

Comment:
I noticed render gets call a lot because hass changes very often so I added this guard as well